README.md

```diff
@@ -22,4 +22,14 @@ Take a look [in the issue tracker](https://github.com/GoogleChrome/web.dev/issue
 
 ## Previewing the site
 
-Unfortunately right now we are only able to open source the content for the site, we're still working on making the rest of the infrastructure public so folks can test it locally. We hope to have a proper staging environment available in the near future. In the meantime, we appreciate your patience 🙏
+The content for web.dev is open source here in this repo.  To create/edit content
```

Comment:
I wonder if it's worth making it very explicit at the top of the README that currently this repo is only for the _written_ content on the site—the actual front-end (CSS, JS) is not yet open source.